app/src/main/java/com/example/firstandroid/presentation/get_bathroom/GetBathroomScreen.kt

```diff
@@ -97,29 +99,34 @@ fun GetBathroomDialog(
                 }
             }
             if (isUpdateBathroomDialogOpen) {
+                updateBathroomViewModel.state.title = selectedBathroomTitle
+                updateBathroomViewModel.state.location = selectedBathroomLocation
+                updateBathroomViewModel.state.cost = selectedBathroomCost
+                updateBathroomViewModel.state.free = selectedBathroomFree
+                updateBathroomViewModel.state.capacity = selectedBathroomCapacity
+                updateBathroomViewModel.state.hours = selectedBathroomHours
+                updateBathroomViewModel.state.id = selectedBathroomId
                 UpdateBathroomDialog(
-                    onDismissRequest = { },
-                    viewModel = updateBathroomViewModel,
-                    originalBathroomInfo = UpdateBathroomState(
-                        id = selectedBathroomId,
-                        title = selectedBathroomTitle,
-                        location = selectedBathroomLocation,
-                        capacity = selectedBathroomCapacity,
-                        free = selectedBathroomFree,
-                        cost = selectedBathroomCost,
-                        hours = selectedBathroomHours
-                    ),
+                    onDismissRequest = { isUpdateBathroomDialogOpen = false},
+                    viewModel = updateBathroomViewModel
                 )
             }
-            Button(
-                onClick = {
-                    isAddBathroomDialogOpen = true
-                },
-                modifier = Modifier
-                    .padding(start = 16.dp, end = 16.dp)
-            ) {
-                Text(text = "Add")
-            }
+        }
+        Button(
+            onClick = {
+                isAddBathroomDialogOpen = true
+            },
+            modifier = Modifier
+                .padding(start = 16.dp, end = 16.dp)
+                .align(Alignment.CenterHorizontally)
+        ) {
+            Text(text = "Add")
+        }
+
```

Comment:
Not sure we need a new ViewModel passed in here, or we could just pass the one from above.